aws/resource_aws_lb_target_group_attachment_test.go

```diff
@@ -3,14 +3,13 @@ package aws
 import (
 	"errors"
 	"fmt"
-	"strconv"
-	"testing"
-
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/elbv2"
 	"github.com/hashicorp/terraform/helper/acctest"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/terraform"
+	"strconv"
```

Comment:
Nit: Same here -- please revert this change. The preferred style for Go `imports` is a section of all standard library imports at the top, a new line, and section(s) of third party imports. Each section alphabetized. See also [Go CodeReviewComments](https://github.com/golang/go/wiki/CodeReviewComments#imports) and tooling like `goimports` that will automatically handle this in your editor. 👍